### DISCUSSION TITLE
DISCORD

### DISCUSSION THREAD

**Discussion by shawnpinciara** [shorepine/amy#133] [Q&A] [answered]
The link is not valid anymore :)

**Comment by bwhitman** [comment]
Hi, can you tell me which link you found that's not valid anymore? 

The correct one is https://discord.com/invite/TzBFkUb8pG

**Reply by shawnpinciara** [reply]
![immagine](https://github.com/shorepine/amy/assets/77097298/fb59c298-5424-4b60-91ab-5c8361351249)
On the homepage: https://github.com/shorepine/amy

**Reply by shawnpinciara** [reply]
The one you sent isn't valid anymore tho..

**Reply by bwhitman** [reply, marked as answer]
This link https://discord.com/invite/TzBFkUb8pG works for me, and many people have joined the discord from that link recently. Are you able to join any other discords? It may be blocked for some reason? Can you share what you see when you click the link?
